File: QuizApp/src/QuizApp.BLL/Dto/ResultAnswer/ResultAnswerDetailDto.cs
using System;
using System.Collections.Generic;
using System.Text;

using QuizApp.BLL.Dto.ResultAnswerOption;

namespace QuizApp.BLL.Dto.ResultAnswer
{
    public class ResultAnswerDetailDto
    {
        public int Id { get; set; }

        public TimeSpan TimeTakenSeconds { get; set; }


        public ICollection<ResultAnswerOptionDetailDto> ResultAnswerOptions { get; set; } = new List<ResultAnswerOptionDetailDto>();
    }
}

File: frontend/src/app/models/result-answer/result-answer-detail-dto.ts
import { ResultAnswerOptionDetailDto } from '../result-answer-option/result-answer-option-detail-dto';
import { ResultQuestionDto } from '../question/result-question-dto';

export interface ResultAnswerDetailDto {
    id: number;
    timeTakenSeconds: string;

    question: ResultQuestionDto;
    resultAnswerOptions: ResultAnswerOptionDetailDto[];
}
